src/components/ui/label.tsx

```
@@ -0,0 +1,7 @@
+import type { LabelHTMLAttributes } from 'react'
+
+type Props = LabelHTMLAttributes<HTMLLabelElement>
+
+export function Label({ className = '', ...props }: Props) {
+  return <label className={`text-sm text-muted-foreground ${className}`} {...props} />
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding an accessible text content to the label element.
Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)